samples/DevTestLabs/Modules/Library/Az.DevTestLabs2.psm1

```diff
@@ -713,6 +713,43 @@ function Set-AzDtlLabSharedImageGallery {
   }
 }
 
+function Get-AzDtlVirtualNetworks {
+  [CmdletBinding()]
+  param(
+    [parameter(Mandatory=$true, ValueFromPipeline=$true, HelpMessage="Lab object to get the Virtual Network from")]
+    [ValidateNotNullOrEmpty()]
+    $Lab,
+
+    [parameter(Mandatory=$false, ValueFromPipeline=$true, HelpMessage="If set, get the Expanded Resource from the std VNet object")]
```

Comment:
Should this be a value from the pipeline?  I think only the Lab would be brought in via the pipeline?